File: 박혜린/3주차/1436.py
n = int(input()) # n은 10,000보다 작거나 같은 자연수. string으로 받는다
count = 0
num = 0

while count != n: 
  num += 1
  if '666' in str(num): # n 번째 count가 나오면 그 수를 출력
    count += 1
  
print(num)

File: 김영진/3주차/1436.c
#include<stdio.h>

int main(void) {
    int n;
    scanf("%d", &n);
    int count = 0;
    int num = 666;

    while (1) {
        int a = 0;
        int cur = num;
        while (cur > 0) {
            int b = cur % 10;
            if (b == 6) {
                a++;
            }
            else {
                a = 0;
            }
            if (a == 3) {
                count++;
                break;
            }
            cur = cur / 10;
        }
        if (count == n) {
            break;
        }
        num++;
    }
    printf("%d\n", num);
}
